package.json

```diff
@@ -18,15 +26,15 @@
         ],
         [
             "microdot/microdot.py",
-            "github:brainelectronics/Micropython-ESP-WiFi-Manager/microdot/microdot.py"
+            "github:miguelgrinberg/microdot/src/microdot.py"
```

Comment:
I would keep the files here with my package until https://github.com/micropython/micropython/pull/11764 is merged and released to avoid breaking changes from the microdot repo. If there are important changes please update the microdot files here. The only change in this file should be the favicon location

Reply:
Ah, you're working on `mip` too... Unfortunately there are two copies of it: one under `mpremote` and one in `micropython-lib/micropython/mip` which is where I've been working. See my PR https://github.com/micropython/micropython-lib/pull/676 . I will add your PR to mine if that's OK with you.